examples/radial_histogram_equalization.py

```diff
@@ -0,0 +1,140 @@
+"""
+===================================
+Radial Histogram Equalization
+===================================
+
+This example applies the radial histogram equalizing filter (`sunkit_image.radial.rhef`) filter to a sunpy map.
```

Comment:
```suggestion
This example applies the Radial Histogram Equalizing (RHE) filter (`sunkit_image.radial.rhef`) filter to a sunpy Map.
```

Reply:
Yeah, I'm never sure if I should keep it as the whole RHEF or just do RHE. The NRGF and FNRGF keep the 'F' at the end though, so I was copying that.

Reply:
I leave that up to you, I am bad at deciding on names.